sapp/ui/issues.py

```diff
@@ -48,6 +49,18 @@
 SinkKindText = aliased(SharedText)
 
 
+class WarningMessageQueryType(graphene.ObjectType):
+    message = graphene.String()
+    code = graphene.Int()
```

Comment:
Instead of creating a new class here, would it be possible to put `message` under `IssueQueryResultType` as `warning_message`?